kubernetes/launch-common.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-# shellcheck disable=SC2034
+# shellcheck disable=SC2034,SC2068
```

Comment:
Is this necessary? (No errors before we added this...?)

Reply:
It is necessary in order to prevent line 19 from throwing a ShellCheck error. And if you obey ShellCheck and put double quotes around the `${@}`, it changes the semantics, causing the Python script to be passed `"cpp python"` as a single string rather than two separate strings as intended

Reply:
OK!